pom.xml

```diff
@@ -438,6 +440,12 @@
                         <exclude>atlassian-ide-plugin.xml</exclude>
                         <exclude>**/PULL_REQUEST_TEMPLATE*</exclude>
                         <exclude>**/new-service-definition-template.xml</exclude>
+                        <exclude>**/swagger/*.js</exclude>
+                        <exclude>**/swagger/*.js.map</exclude>
+                        <exclude>**/swagger/*.html</exclude>
+                        <exclude>**/swagger/*.css</exclude>
+                        <exclude>**/swagger/*.css.map</exclude>
+                        <exclude>**/swagger/*.md</exclude>
```

Comment:
I don't think this is correct. We need license notices on most of our files. Especially the js, html, css, and md. If the filetype supports comments - there should be a license.

Reply:
So basically, as I explained in the PR description above, these files were downloaded from OpenAPI homepage and were unpacked. I did not want to use [npm](https://www.npmjs.com/package/swagger-ui), because I had to Knoxify the `index.html` page.
Do you think we should add our Apache license into these downloaded static files too?

Reply:
It should be the Swagger license whatever that is and it should be added to the downloaded static files. It should not be the Apache license.

Reply:
Actually, Swagger is under Apache 2.0: https://swagger.io/license/
Let me add license headers accordingly.

Reply:
Fixed.